lms/templates/main.html

```diff
@@ -167,6 +179,7 @@
   % endif
 
   <%block name="js_extra"/>
+  <script type="text/javascript" src="${static.url('js/vendor/noreferrer.js')}" charset="utf-8"></script>
```

Comment:
Note to reviewers: This is a fix for a security issue that could leak information on the password reset page to links that had been marked as `rel=noreferrer` in certain browsers.  Moving it here to include it on every LMS page without including it in the HTML sent to the marketing site and blog, which don't have this security vulnerability.